tap-snapshots/test/lib/utils/error-message.js.test.cjs

```diff
@@ -157,10 +160,11 @@ Object {
     ],
     Array [
       "404",
-      String(
-        
-        Note that you can also install from a
-      ),
+      "",
+    ],
+    Array [
+      "404",
+      "Note that you can also install from a",
```

Comment:
These snapshot changes don't actually change what is displayed to the user. The snapshots changed because these are unit tests.

When we format these messages consecutive calls to output with the same first param (`404` in this case) are rendered identically to one call with a message that includes newliens.